src/pages/Organization/OrganizationUsers.tsx

```diff
@@ -149,11 +149,11 @@ export default function OrganizationUsers({ id, navOrganizationId }: Props) {
           />
         </div>
         {isFetchingUsers ? (
-          <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4">
+          <div className="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3">
             <CardGridSkeleton count={6} />
           </div>
         ) : (
-          <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-4">
+          <div className="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3">
```

Comment:
This looks too congested, I'd suggest changing both to lg:grid-cols-2 and xl:grid-cols-3.

Changing that should also allow to discard some of the other changes below 🤔